packages/builder/src/pages/builder/portal/apps/index.svelte

```diff
@@ -33,14 +32,16 @@
   let selectedApp
   let creationModal
   let updatingModal
-  let exportModal
   let creatingApp = false
   let loaded = $apps?.length || $templates?.length
   let searchTerm = ""
   let cloud = $admin.cloud
   let creatingFromTemplate = false
   let automationErrors
   let accessFilterList = null
+  let welcomeHeader, welcomeBody
+  let createAppButtonText
+  let enrichedApps, filteredApps, lockedApps, unlocked
```

Comment:
Since these are all declared as reactive statements, we don't need to define them again as local variables. I don't think doing this will cause bugs, but it might affect how well svelte is able to hoist dependencies since these are now local variables rather than svelte reactive statement declarations.

Basically at best it does nothing, at worst it might cause edge case issues.

Reply:
I might need to look into my version of Webstorm svelte as it seems to see this as a bug - hoisting them helps to make the files parse-able, as without this every file I look at is just a sea of red lol